gemrb/GUIScripts/TextScreen.py

```diff
@@ -194,8 +194,8 @@ def EndTextScreen ():
 def ReplayTextScreen ():
 	global TextArea, TableName, Row
 
-	# stop any current speech, flag of 5 = GEM_SND_RELATIVE|GEM_SND_SPEECH
-	GemRB.PlaySound(None, CHAN_GUI, 0, 0, 5)
+	# stop any current speech, flag of 4 = GEM_SND_SPEECH
+	GemRB.PlaySound(None, CHAN_GUI, 0, 0, 4)
```

Comment:
you can replace it here as well and remove that part of the comment.

Reply:
Urgh, thanks